oemof_flexmex/plotting/prepare.py

```diff
@@ -0,0 +1,168 @@
+            demand = demand + df_demand[(df_demand.loc[:, 'Parameter'] == i) & (df_demand.loc[:, 'Region'] == 'DE')].loc[:, 'Value'].iloc[0]
+            print(demand)
+    elif onxaxes == 'Region':
+        for region in ('AT', 'BE', 'CH', 'CZ', 'DE', 'DK', 'FR', 'IT', 'LU', 'NL', 'PL'):
+            plot_data = sum_transmissions(plot_data, 'FlexMex2_2c', region)
+        demand = 0
+    df_plot_conversion_electricity = pd.crosstab(index=plot_data[onxaxes], columns=plot_data.Parameter,
+                                           values=plot_data.Value, aggfunc='mean')
+#    df_plot_conversion_electricity = \
+#            df_plot_conversion_electricity.reindex(columns=['EnergyConversion_SecondaryEnergy_Electricity_CH4_GT',
+#                                                            'EnergyConversion_SecondaryEnergy_Electricity_RE',
+#                                                            'Transmission_Incoming',
+#                                                            'EnergyConversion_SecondaryEnergy_Electricity_Slack',
+#                                                            'EnergyConversion_Curtailment_Electricity_RE',
+#                                                            'Transmission_Losses',
+#                                                            'Transmission_Outgoing'])
+    return df_plot_conversion_electricity, demand
+
+def conversion_heat_FlexMex2_2(plot_data, df_demand, onxaxes):
+    plot_data = onxaxes_preparation(plot_data, onxaxes, 'FlexMex2_2c')
+    plot_data.to_csv('2021-07-03_plot_data.csv')
+
+    parameters = ['EnergyConversion_SecondaryEnergy_Heat_Electricity_Large',
+                  'EnergyConversion_SecondaryEnergy_Heat_ElectricityHeat_CH4_ExCCGT',
+                  'EnergyConversion_SecondaryEnergy_Heat_ElectricityHeat_Large',
+                  'EnergyConversion_SecondaryEnergy_Heat_ElectricityHeat_Small',
+                  'EnergyConversion_SecondaryEnergy_Heat_Gas_Large',
+                  'EnergyConversion_SecondaryEnergy_Heat_Slack',
+                  ]
+    plot_data = plot_data.loc[plot_data['Parameter'].isin(parameters)]
+    # sum all outgoing and all ingoing transmissions for each scenario
+
+    demand = 0
+    if onxaxes == 'Scenario':
+        for i in ['Energy_FinalEnergy_Heat_CHP', 'Energy_FinalEnergy_Heat_HeatPump']:
+            demand = demand + df_demand[(df_demand.loc[:, 'Parameter'] == i) & (df_demand.loc[:, 'Region'] == 'DE')].loc[:,
+                          'Value'].iloc[0]
+
+    df_plot_conversion_heat = pd.crosstab(index=plot_data[onxaxes], columns=plot_data.Parameter,
+                                           values=plot_data.Value, aggfunc='mean')
+#    df_plot_conversion_electricity = \
+#            df_plot_conversion_electricity.reindex(columns=[])
+    return df_plot_conversion_heat, demand
+
+def storage_FlexMex2_2(plot_data, onxaxes):
+    plot_data = onxaxes_preparation(plot_data, onxaxes, 'FlexMex2_2c')
+    parameters = [
```

Comment:
I would suggest to move these parameters outside of the function. It could be defined in the script.